docs/source/demos/text_input.ipynb

```diff
@@ -0,0 +1,343 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we can score the pipeline on the hold out data using the core objectives for binary classification problems"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "best_pipeline.score(X_holdout, y_holdout, objectives=evalml.objectives.get_core_objectives('binary'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As you can see, this model performs relatively well on this dataset, even on unseen data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Why encode text this way?\n",
+    "\n",
+    "To demonstrate the importance of text-specific modeling, let's train a model with the same dataset, without letting `AutoMLSearch` detect the text column. We can change this by explicitly setting the data type of the `'Message'` column in Woodwork."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from woodwork import DataTable\n",
+    "\n",
+    "X_train_datatable = DataTable(X_train, logical_types={'Message': 'Categorical'})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "automl_no_text = AutoMLSearch(problem_type='binary',\n",
+    "                      max_iterations=5,\n",
+    "                      optimize_thresholds=True)\n",
+    "\n",
+    "automl_no_text.search(X_train_datatable, y_train)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Like before, we can look at the rankings and pick the best pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "automl_no_text.rankings"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "best_pipeline_no_text = automl_no_text.best_pipeline\n",
+    "best_pipeline_no_text.graph()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here, changing the data type of the text column removed the `Text Featurization Component` from the pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "automl_no_text.describe_pipeline(automl_no_text.rankings.iloc[0][\"id\"])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# train on the full training data\n",
+    "best_pipeline_no_text.fit(X_train, y_train)\n",
+    "\n",
+    "# get standard performance metrics on holdout data\n",
+    "best_pipeline_no_text.score(X_holdout, y_holdout,  objectives=evalml.objectives.get_core_objectives('binary'))"
```

Comment:
Could you please do
```python
scores = best_pipeline_no_text.score(X_holdout, y_holdout,  objectives=evalml.objectives.get_core_objectives('binary'))
print(f'Accuracy Binary: {scores['Accuracy Binary']}')
```
That'll mesh well with your paragraph below where you call out the accuracy being 75% here.